uni/ulsp/database.icn

```diff
@@ -96,14 +96,15 @@ class LSPDB(
    end
 
    method build_package_db()
-      local dirPath, db_paths
+      local dirPath, db_paths := [], dir
 
       dirPath := unicon_dir
 
-      db_paths := ipaths_get()
-
-      pop(db_paths) # remove the current directory
-
+      write("Looking in:")
+      every dir := !ipaths_get() do
+         # remove the current dir, also skip pasrer due to a unidoc bug
+         if not((dir == ("." | "")) | find("parser", dir))then
```

Comment:
I think this might be a bit too sweeping. It removes any path with "parser" somewhere in it.  I think I'd prefer "uni/parser"

Reply:
fixed